src/Silverback.Integration/Messaging/Validation/MessageValidator.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace Silverback.Messaging.Validation
+{
+    /// <summary>
```

Comment:
@ChrisBerna summaries are mandatory only for the public API. You don't need to write them religiously for the internal types.

Reply:
@BEagle1984 ok good to know. 😄